actions/src/main/kotlin/com/alfresco/content/actions/CreateFolderFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.alfresco.content.actions
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.setFragmentResultListener
+import com.alfresco.content.withFragment
+import kotlinx.coroutines.CancellableContinuation
+import kotlinx.coroutines.suspendCancellableCoroutine
+
+class CreateFolderFragment : Fragment() {
```

Comment:
You actually don't need this fragment to show the dialog and return the result.
You can use the same pattern we use for `MaterialAlertDialogBuilder`.
So add `onSuccess(title: String, description: String)` on `CreateFolderDialog` and then wrap it in a continuation same as the regular dialog.

Reply:
Done